src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-description.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>This description will help users identify this token when setting up build triggers</p>
```

Comment:
Technically incorrect. "This description will help administrators identify this token when configuring a Bitbucket server instance"

Reply:
@ctan-atlassian just going to make this change